tests/test_copydb.py

```diff
@@ -110,15 +129,18 @@ def test_copydb_template_not_exists_target_exists():
         _dropdb(TEST_DBNAME_NEW)
 
 
-def test_copydb_no_source_filestore(pgdb):
+def test_copydb_no_source_filestore(odoodb):
     filestore_dir_new = odoo.tools.config.filestore(TEST_DBNAME_NEW)
+    filestore_dir_original = odoo.tools.config.filestore(odoodb)
+    if os.path.exists(filestore_dir_original):
+        shutil.rmtree(filestore_dir_original)
```

Comment:
Do you remember why this rmtree was necessary?

Reply:
We now use `odoodb` fixture in place of `pgdb` (We need a properly initialized Odoo db to check if settings are reset) . Therefore a filestore is created by default in odoo => 10.